CONTRIBUTING.md

```diff
@@ -1225,7 +1225,13 @@ $ yarn test
 ## Changing Cloud Assembly Schema
 
 If you plan on making changes to the `cloud-assembly-schema` package, make sure you familiarize yourself with
-its own [contribution guide](./packages/@aws-cdk/cloud-assembly-schema/CONTRIBUTING.md)
+its own [contribution guide](https://github.com/cdklabs/cloud-assembly-schema/blob/main/CONTRIBUTING.md).
+
+The team will not actively monitor the [cloud-assembly-schema](https://github.com/cdklabs/cloud-assembly-schema)
+repository for contributions. If you have a PR that needs to modify `cloud-assembly-schema`, open both PRs and
+ensure that you link the `cloud-assembly-schema` PR in the one opened against this repository.
+The `cloud-assembly-schema` changes must be merged (and released) before the PR in this repository can be merged.
+The PR in this repository must be updated to use the new release of the `cloud-assembly-schema` before it can be merged.
```

Comment:
oh wait we don't automatically grab new versions of cloud-assembly-schema when they're released? we have to manually update the version?

a) if thats the case, is that safe from a security perspective
b) we should probably have an example/instruction for what the PR against aws-cdk repo looks like then

Reply:
We do grab new ones, it's just reliant on dependabot. I can clarify that we either have to wait for the version bump or manually bump it, but the point was that it's been approved, but it can't merge until dependabot decides to update it.